api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1011,6 +1011,8 @@ type GlobalConfig struct {
 	// +listType=set
 	Tags []string `json:"tags,omitempty"`
 
+	Env []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
Could we please add comment there explaining what this new option does ? This will appear in `docs/configuration.v2alpha1.md` which is currently empty